src/App.js

```diff
@@ -1,13 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
 
-import Bookings from "./Bookings";
 import "./App.css";
 
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Bookings from "./Bookings";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+import NewBookingsForm from "./NewBookingsForm";
+
+const hotelInfos = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+
 const App = () => {
+  const [newBooking, setNewBooking] = useState("");
+
+  function getNewBooking(booking) {
+    setNewBooking(booking);
+  }
```

Comment:
I'm not sure this is exactly what we want. At the moment, I think we can only add 1 booking. If you try to add a second booking, nothing happens.

What could we do to fix that? Hint: what we actually want to do is add the new booking to our existing list of bookings. So how could we do that?

Reply:
@40thieves Many thanks for the feedback, I updated it and it works now the way it should be.